pandas-stubs/core/frame.pyi

```diff
@@ -140,7 +140,8 @@ class _LocIndexerFrame(_LocIndexer):
             | MaskType
             | slice
             | list[HashableT]
-            | tuple[str | int | slice, ...],
+            | tuple[str | int | slice, ...]
+            | Hashable,
```

Comment:
If we have `Hashable` here, we probably don't need `tuple[...]` and `slice` as they should also be hashable.

Reply:
OK, fixed in next commit.